dockbix-agent-xxl-daemonset.yaml

```diff
@@ -19,18 +19,20 @@ spec:
       hostIPC: true
       hostNetwork: false
       containers:
-        - resources:
-            requests:
-              cpu: 0.15
+        - image: monitoringartist/dockbix-agent-xxl-limited
+          name: dockbix
           securityContext:
-            privileged: true
+             privileged: true
+          resources:
+            requests:k
```

Comment:
Is it correct syntax `requests:k`?

Reply:
k is wrong thing happen while saving the file, i have reset